lib/pure/distros.nim

```diff
@@ -195,6 +199,8 @@ proc detectOsImpl(d: Distribution): bool =
         result = ("sun" in uname) or ("solaris" in uname)
       of Distribution.Haiku:
         result = defined(haiku)
+      of Distribution.Void:
+        result = "void" in osReleaseID()
```

Comment:
Why not just add this to the branch above along with Fedora, Elementary, etc? It will work just as well

Reply:
Curious to why "arch" is split out. I wasn't sure of the specifics, so I added it that way. If there is no great reason, I can just squash them all in (rhel, arch, artix, void) to be up with the rest.

Reply:
because it would check for "archlinux" which isn't the same as "arch", no?

Reply:
Good point. Resolving your requested change. Thank you for your feedback to something that was obvious.